CHANGES.md

```diff
@@ -28,6 +28,10 @@ breaking changes, and mappings for the large list of deprecated functions.
 
 [Migration guide]: https://github.com/openssl/openssl/tree/master/doc/man7/migration_guide.pod
 
+### Changes between 3.0 and 3.1 [xx XXX xxxx]
```

Comment:
Similar to the other comment: https://github.com/openssl/openssl/pull/16484/files#r699715510